diagnostics-app/src/app/shared/header/header.component.ts

```diff
@@ -32,6 +38,14 @@ export class HeaderComponent {
     this.toggleDrawer.emit();
   }
 
+  public onToggleModeChange(event: MatSlideToggleChange) {
+    if (event.checked) {
+      this.router.navigate(['/support-assist']);
```

Comment:
`this.router.navigateByUrl('/support-assist')`?

Reply:
Done.